examples/widgets/painter.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import print_function
+"""
+Drag the mouse to paint selected areas of the plot. The callback prints
```

Comment:
Can you add a title and such to match gallery stuff (see [guidelines](https://github.com/matplotlib/matplotlib/blob/master/doc/devel/MEP/MEP12.rst#clean-up-guidelines))?